Resources/Locale/en-US/singularity/components/containment-component.ftl

```diff
@@ -10,4 +10,7 @@ comp-containment-disconnected = The containment field generator light fades away
 comp-containment-anchor-warning = You cannot unanchor the containment field generator while it's on or connected!
 comp-containment-toggle-warning = You cannot turn the generator off while it's connected.
 
+comp-containment-alert-field-link-broken = Containment field connection {$location} has broken.
+comp-containment-alert-field-losing-power = Containment field generator {$location} will soon not have enough power to maintain any connections.
```

Comment:
These messages could probably be more uniform with the popup text or shorter. The location text is variable length so it should probably go at the end of the message (every other location message I quickly searched also does this).
```suggestion
comp-containment-alert-field-link-broken = Containment field connection has severed {$location}.
comp-containment-alert-field-losing-power = Containment field generator is losing power {$location}.
```